CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Fixed
+- Fix `quote_style` Bug: Rufo breaks HEREDOC syntax. (issue [#88](https://github.com/ruby-formatter/rufo/issues/88)). This fixes heredoc within parens formatting causing bad formatting and subsequent syntax error.
```

Comment:
What does this change have to do with `quote_style`? Was the bug introduced by the change that added it?

Reply:
Oops ~ I thought `quote_style` was some fancy markdown formatting that was being used in the changelog (cut-paste error) :blush: I'll fix that